client/src/app/tabs/cloud-bpmn/BpmnEditor.js

```diff
@@ -245,6 +249,22 @@ export class BpmnEditor extends CachedComponent {
     });
   }
 
+  handleEngineProfileChange = ({ engineProfile }) => {
+    const { executionPlatformVersion: version } = engineProfile;
+    if (!version) return;
+
+    const elementTemplates = this.getModeler().get('elementTemplates');
+
+    const engines = {
+      ...elementTemplates.getEngines(),
+      camunda: version
+    };
+
+    elementTemplates.setEngines(engines);
+
+    this.loadTemplates();
```

Comment:
This should not be necessary, or should it? Templates update automatically when the engine changes.

Reply:
I'd expect that it works out of the box without additional calls.